pkgs/development/tools/build-managers/bazel/bazel-remote/default.nix

```diff
@@ -64,11 +67,19 @@ buildBazelPackage rec {
       sed -e '/^FILE:@bazel_gazelle_go_repository_tools.*/d' -i $bazelOut/external/\@*.marker
     '';
 
-    sha256 = "1m7fmb03lirffxx04ck73bn5zwaji7zdwhlqq8s1c6pgp755d3vi";
+    sha256 = if stdenv.isDarwin then "0h6hmm8kkkwk0q8idlbfr50iplmrm2g7dsy6f5k2hilp8xmfssmw" else "1b6252cz26qwc5nvd34ld3fr908bz55w136x9y27yx39z5ynnbb8";
```

Comment:
This makes me think there is some non-deterministic dependency, probably some native files from protobuf being built.

Reply:
Pretty sure this is not gonna work if anybody ever has to build this fixed output instead of downloading it from the cache, but I don’t have enough stakes in this game to invest time fixing it :)